tests/fixtures/input-plugins/input-plugin-checkbox.js

```diff
@@ -33,5 +33,5 @@ module.exports = {
       },
     },
   },
-  html: `<label for="{{checkboxSelector.id}}">{{checkboxSelector.label}}</label>{% for option in checkboxSelector.options %}<label><input type="{{checkboxSelector.type}}" name={{checkboxSelector.name}} id="{{checkboxSelector.id}}" value="{{checkboxSelector.value}}" {% if option.value == checkboxSelector.value %}checked{% endif %}>{{option.label}}</label>{% endfor %}`,
-};
\ No newline at end of file
+  html: `{% for option in checkboxSelector.options %}<label for="{{checkboxSelector.id}}--{{option.index}}"><input type="{{checkboxSelector.type}}" name="{{checkboxSelector.name}}" id="{{checkboxSelector.id}}--{{option.index}}" value="{{option.value}}" {% if option.value == checkboxSelector.value %}checked{% endif %}>{{option.label}}</label>{% endfor %}`,
```

Comment:
@Snugug This is what I've worked up for the fixture:

```
  html: `{% if checkboxSelector.options.length > 1 %}<fieldset id="{{checkboxSelector.id}}"><legend>{{checkboxSelector.label}}</legend>{% endif %}
    {% for option in checkboxSelector.options %}<label for="{{checkboxSelector.id}}--{{option.index}}">
    <input type="{{checkboxSelector.type}}" name="{{checkboxSelector.name}}" id="{{checkboxSelector.id}}--{{option.index}}" value="{{option.value}}" {% if option.value == checkboxSelector.value %}checked{% endif %}>{{option.label}}</label>{% endfor %}
    {% if checkboxSelector.options.length > 1 %}</fieldset>{% endif %}`,
```

Which would then translate to be the code for the actual plugin...if it works for you.